themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -138,16 +138,23 @@
       <div class="{{ container }}">
         {{ page.footer_main }}
         {% if page.footer_first or page.footer_second or page.footer_third or page.footer_fourth %}
-          <div class="site-footer__top clearfix">
-            {{ page.footer_first }}
-            {{ page.footer_second }}
-            {{ page.footer_third }}
-            {{ page.footer_fourth }}
-          </div>
-        {% endif %}
-        {% if page.footer_fifth %}
-          <div class="site-footer__bottom">
-            {{ page.footer_fifth }}
+          <div class="site-footer-blocks row">
+            {% for i, region in filled_regions %}
+              {% set classes = '' %}
```

Comment:
We could use Drupal `Attributes` here - usage of `<div class="{{ classes }}">` is discouraged in D8.
It works like this: https://www.drupal.org/docs/8/theming-drupal-8/using-attributes-in-templates